Files/ViewModels/SettingsViewModel.cs

```diff
@@ -53,6 +68,26 @@ public SettingsViewModel()
             {
                 DefaultLanguages.Add(new DefaultLanguageModel(lang));
             }
+
+            return this;
+        }
+
+        public static Task<SettingsViewModel> CreateInstance()
+        {
+            var settings = new SettingsViewModel(false);
+            return settings.Initialize();
+        }
+
+        private SettingsViewModel() : this(true)
+        {
+        }
+
+        private SettingsViewModel(bool initialize)
```

Comment:
At the end I'd keep only the constructor that's actually used

Reply:
Removed